tests/ceph_ansible/switch_rpm_to_container.py

```diff
@@ -1,21 +1,78 @@
 """switches non-containerized ceph daemon to containerized ceph daemon"""
+import yaml
+
 from utility.log import Log
 
 log = Log(__name__)
 
 
+def ceph_repository_type_cdn(ansible_dir, installer_node):
+    out, err = installer_node.exec_command(
+        sudo=True,
+        cmd="cat {ansible_dir}/group_vars/all.yml.sample".format(
+            ansible_dir=ansible_dir
+        ),
+    )
+    sample_conf = yaml.safe_load(out)
+    docker_registry = sample_conf.get("ceph_docker_registry")
+    docker_image = sample_conf.get("ceph_docker_image")
+    docker_image_tag = sample_conf.get("ceph_docker_image_tag")
+
+    return {
```

Comment:
We don't need to return dict here, since we are not receiving same in calling method.

`return docker_registry, docker_image, docker_image_tag`